store.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Karma Store</title>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style2.css">
+</head>
+
+<body>
+    <div class="contener">
+        <div class="nav1">
+            <nav>
+                <img src="img/karma-logo.svg" class="logo">
+                <ul class="nav_bar">
+                    <a href="index.html" class="home"> <strong>Meet Karma</strong></a>
+                    <a href="#" class="home">How it Works</a>
+                    <a class="home" id="active" href="#" class="home">Store</a>
+                    <a href="#" class="home">Blog</a>
+                    <a href="#" class="home">Help</a>
+                    <a href="#" class="home">Login</a </ul>
+            </nav>
+        </div>
+        <div class="form1">
+
+            <h1>Order your Karma wifi device today!</h1>
+            <form>
+                <div class="f">
+                    <label for="username1">First name *</label>
+                    <label for="username2">Last name *</label>
```

Comment:
You should not seperate the label form their input, instead the common use of the label is to have it next to its input like this:
 ``` 
 <div>
 <label for="username1">First name *</label>
 <input id="username1" type="text" name="username" />
 </div>